style.css

```diff
@@ -23,48 +25,129 @@ a {
   text-decoration: none;
 }
 
-header {
+/* header {
   position: fixed;
   top: 0%;
   z-index: 5;
   box-shadow: 0.1rem 0.1rem rgba(245, 243, 243, 0.918);
   background: #fff;
-  overflow: hidden;
-  background-color: #fff;
-  width: 100%;
-  height: 916px;
+} */
+
+.navbar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
 }
 
-.header {
+.right-nav {
+  color: black;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
+  position: fixed;
+  left: -100%;
+  top: 70px;
+  flex-direction: column;
+  background-color: #5ac057;
+  width: 400px;
+  text-align: center;
+  transition: 0.3s;
+}
```

Comment:
I am not quite sure what happened, but both your mobile and desktop designs are out of whack.

Mobile design:

<img width="185" alt="Screen Shot 2022-06-21 at 7 51 25" src="https://user-images.githubusercontent.com/82426682/174803468-f7afecbc-8c63-4688-9bf9-95dfc736ba0e.png">

Desktop design:

<img width="1440" alt="Screen Shot 2022-06-21 at 7 52 07" src="https://user-images.githubusercontent.com/82426682/174803620-351c3396-50df-4ede-ae2b-61bf4e34e534.png">

Please fix them to make them match the template design. I am using Google Chrome, for reference.

Additionally, your menu doesn't match the template design.

<img width="213" alt="Screen Shot 2022-06-21 at 7 53 20" src="https://user-images.githubusercontent.com/82426682/174803872-ad0b7474-4ac0-49cd-87ff-49d0aa351d29.png">

Please fix it as well.

<img width="844" alt="Screen Shot 2022-06-21 at 7 53 52" src="https://user-images.githubusercontent.com/82426682/174803995-bcbdb717-0658-4c36-ac55-997d86d97ed8.png">